superbench/config/azure_ndmv4.yaml

```diff
@@ -81,7 +81,7 @@ superbench:
           prefix: CUDA_VISIBLE_DEVICES={proc_rank} numactl -N $(({proc_rank}/2))
           parallel: no
     disk-benchmark:
-      enable: true
+      enable: false
```

Comment:
update disk-benchmark in superbench/config/azure_ndv4.yaml accordingly

Reply:
ok, updated